pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -209,6 +214,40 @@ public List<Map<String, Object>> getServerToSegmentsMap(
     return resultList;
   }
 
+  @GET
+  @Path("segments/{tableName}/xdebug")
```

Comment:
perhaps have a more meaning method name

Reply:
Yeah, I'll fix this in future iterations.. there's quite a lot of refactoring still pending.